src/xcode/ENA/ENA/Source/Client/Client.swift

```diff
@@ -38,15 +38,69 @@ protocol Client {
 	/// Gets the app configuration
 	func appConfiguration(completion: @escaping AppConfigurationCompletion)
 
+	#if INTEROP
+	/// Determines days that can be downloaded.
+	///
+	/// - Parameters:
+	///   - country: Country code
+	///   - completion: completion callback which includes the list of available days
+	func availableDays(
```

Comment:
country code in which format/standard? i have seen that we use this also in other PRs, it could make sense to introduce an enum here? Just a thought, maybe for the future.

Reply:
We could also create a `typealias` to the `id` field in `Country` and use this. Then it's clear and one knows where to get this identifier from.

Reply:
Isn't the list of countries "dynamic"?

Reply:
good point, so enums are probably not the way to go! How is this handled throughout the other parts of the app? Do we fetch flags and all other country information from the server or do we have a mix between dynamic and static content here?

Reply:
We have a mix. We have static content with all countries. But the backend decides which of these countries are "unlocked".